pkg/config/setup/system_probe_cws.go

```diff
@@ -149,3 +150,106 @@ func initCWSSystemProbeConfig(cfg pkgconfigmodel.Config) {
 
 	cfg.BindEnvAndSetDefault("runtime_security_config.network_monitoring.enabled", false)
 }
+
+var (
```

Comment:
This list would end up in all the agent metadata payload of the Agent and in REDAPL. This would be a significant increase in size.

Do we need this in the configuration ? Could we define a empty slice here and move the list to the system-probe code ? how likely are users to edit this list ?

Reply:
No worries, I moved it to the `system-probe` package. It's more of a safeguard in case we missed something, or if anything goes wrong with this feature.